testing/integration-tests/copy_resources.rs

```diff
@@ -0,0 +1,27 @@
+extern crate fs_extra;
+use fs_extra::dir::copy;
+use fs_extra::dir::*;
```

Comment:
Please avoid wildcard imports except from preludes and the like.
This code is not "productoin", so this is more guidance on good style.

Reply:
Thanks. Every code should have a production quality, even test code. So don't hesitate to put more remarks about its quality